src/app/containers/manuscript/styles.scss

```diff
@@ -1,9 +1,22 @@
-
 .manuscript-container {
-  padding: 10px;
+  margin: 0px;
+  padding: 0px;
 
   .manuscript-field {
     padding: 20px 0;
   }
 }
 
+.manuscript-toolbar {
+  width: 100%;
+  min-height: 64px;
+  margin: 0px;
+  background-color: #fafafa;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.2);
```

Comment:
I can't find any docs for this syntax. Does it set shadow for 3 sides? or is it 3 overlapping shadows?